src/components/Avatar.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="group relative w-24 h-24">
+    <img v-if="imgPath" :src="imgPath" alt="" class="w-24 h-24 rounded-full" />
+    <div
+      v-if="!imgPath"
+      class="
+        w-24
+        h-24
+        relative
+        flex
+        justify-center
+        items-center
+        rounded-full
+        bg-gray-500
+        text-6xl text-white
+        uppercase
+      "
+    >
+      {{ giveName() }}
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    firstName: String,
+    imgPath: String,
```

Comment:
rozszerz propsy - https://v3.vuejs.org/guide/component-props.html#prop-validation
wpisz czy required, a jeśli nie to wartość defalut

Reply:
w sumie nawet fajnie jak wyswietli sie jakis default avatar :D

Reply:
default to ogólnie zawsze